docs/reference/code/readme

```diff
@@ -0,0 +1,106 @@
+# Testing documented authd deployment script
```

Comment:
Adding a `.md` suffix to the filename would allow us to view the rendered markdown in GitHub. Without that extension, GitHub only shows it in plaintext: https://github.com/ubuntu/authd/blob/cca3f8ac974a2d3a9b5b7e505de63890fb410d12/docs/reference/code/readme